apps/src/craft/code-connection/cc-client.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * CC client for 3rd party extension
+ *
+ */
+
+export default class cc_client {
+    constructor(port) {
+        this.port = port;
+        this.baseUrl = "http://localhost:";
```

Comment:
Ah never mind, we can change the port in the constructor, and any tests will likely mock at a different level.  Sorry!